.github/ISSUE_TEMPLATE/1_bug_report.yaml

```diff
@@ -32,11 +32,10 @@ body:
       description: select all version where you have experienced this issue
       multiple: true
       options:
-        - "v1.6"
-        - "v1.7"
```

Comment:
Shall we have one aggregate label `legacy` for the versions before?

Reply:
@Borda feel free to send a PR if you think it is good to have. I have no preference on this. As you wish